### ISSUE TITLE
Slow GPU and CPU Performance

### ISSUE THREAD

**Issue by tvieira20** [Jie2GG/Hackintosh-ASUS-S5300FN#15]
Hi, I generated a custom SSDT, moved it into OC folder and enabled it. For a few days it performed really well, but now it's really laggy. What else can I do? 
Can you upload any newer EFI folder so I can try it out and see if it fixes anything that I've may messed up?

Vivobook S530FN with the i7-8565U Whiskey Lake + UHD 620
Thanks and Regards

**Comment by Jie2GG** [comment]
OK, I ’m upload new files. you can use any files of EFI

**Comment by Jie2GG** [comment]
But these files no solve issues of CPU and GPU. My laptop in long sleep will lead to CPU Core frequency low

**Comment by Jie2GG** [comment]
In normal situation, CPU Core frequency in 2.8GHz (Windows OS), I try changed CPU Fan RPM, But seem no success of improve CPU Core frequency (Exclude CPU heat dissipation issues).

**Comment by tvieira20** [comment]
So to update: I generated... AGAIN a Power Management folder using CPUFriendFriend. 
There was an option that I have never seen before called "Properties for Power Saving" and I enabled it. 
<img width="697" alt="Captura de ecrã 2021-06-23, às 10 58 08" src="https://user-images.githubusercontent.com/55699311/123234695-1165bc00-d4d3-11eb-8330-6a7424a305d5.png">
Moved the generated files to EFI and went to OC Configurator and Enabled CPUFriend.kext and CPUFriendDataProvider.kext.
A reboot and now my laptop is way better. GPU transitions are more smooth and it doesn't lag at all. That option was probably what it was missing. 
Benchmark shows the same results as before, but GPU is more smooth.
Regards mate
